hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdiff/TestRocksDBCheckpointDiffer.java

```diff
@@ -412,20 +436,21 @@ void diffAllSnapshots(RocksDBCheckpointDiffer differ)
     // Hard-coded expected output.
     // The results are deterministic. Retrieved from a successful run.
     final List<List<String>> expectedDifferResult = asList(
-        asList("000024", "000017", "000028", "000026", "000019", "000021"),
-        asList("000024", "000028", "000026", "000019", "000021"),
-        asList("000024", "000028", "000026", "000021"),
-        asList("000024", "000028", "000026"),
-        asList("000028", "000026"),
-        Collections.singletonList("000028"),
+        asList("000023", "000029", "000026", "000019", "000021", "000031"),
```

Comment:
This test is changed because of addition of new column family: `compactionLogTable`.